vehicle/homeassistant.go

```diff
@@ -173,6 +174,9 @@ func (v *HomeAssistant) getFloatSensor(entity string) (float64, error) {
 		return 0, err
 	}
 
+	// Replace comma with dot for locales that use comma as decimal separator
+	s = strings.ReplaceAll(s, ",", ".")
```

Comment:
@thecem does HA deliver JSON or plain text? If the latter- is JSON possible? Then we could avoid this ambiguity.

Reply:
no, we use the same standardized api that sends (normaly) valid json, but (and this is the main problem) it could also send string ect. I tested this with 322(OK) / 322.456(OK) / 323,345 (NOK -> Last version throws an error in evcc log and switches off the rng km in UI).

https://developers.home-assistant.io/docs/api/rest/#actions

"The API accepts and returns only JSON encoded objects."

Reply:
Open ticket with ha?

Reply:
I repeat the question? Does HA have a JSON api? Where a number is returned as such?

Reply:
"The API accepts and returns only JSON encoded objects." There is only one REST API,

with : “323,456“ (string); 323.456 float; 323 int